easybuild/tools/run.py

```diff
@@ -108,7 +108,7 @@ def cache_aware_func(cmd, *args, **kwargs):
 
 @run_cache
 def run(cmd, fail_on_error=True, split_stderr=False, stdin=None, env=None,
-        hidden=False, in_dry_run=False, work_dir=None, shell=True,
+        hidden=False, in_dry_run=False, verbose_dry_run=False, work_dir=None, shell=True,
```

Comment:
having both `hidden` (to disable output in both trace and dry run mode) and `verbose_dry_run` is necessary evil, since there are situations where we do not want trace output, but we do want to see commands that would be executed in dry run mode (primarily in `get_source_tarball_from_git`)